src/SI_Toolkit/Predictors/predictor_wrapper.py

```diff
@@ -0,0 +1,136 @@
+import os
+import yaml
+from copy import deepcopy as dcp
+
+# predictors config
+predictors_config = yaml.load(open(os.path.join('SI_Toolkit_ASF', 'config_predictors.yml'), 'r'), Loader=yaml.FullLoader)
```

Comment:
Nice, this is starting to shape up. I think it's the right place to read from config.

At first I was sceptical whether we should split up the config like this. However, as long as they are named differently it works. And I like how you structured this.